src/PageContent/Directory/DirectoryPage.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import DirectoryCard from './DirectoryCard.js';
+import DirectoryFilter from './DirectoryFilter.js';
+import './Directory.css';
+import "typeface-roboto";
+
+// Component for directory card
+
+class DirectoryPage extends Component {
+
+    render() {
+		return (
+            <div className="directoryPage">
```

Comment:
nity grity but line 13 is not indented properly under return

Reply:
return itself isn't indented correctly either